ui.frontend/src/view/FormFieldBase.js

```diff
@@ -78,6 +92,60 @@ export default class FormFieldBase extends FormField {
         }
         this._updateVisible(state.visible)
         this._updateEnable(state.visible)
+        this._initializeHelpContent();
+    }
+
+    _initializeHelpContent() {
+        this._showHideTooltipDiv(false);
+        this._showHideLongDescriptionDiv(false);
+        if (this.getTooltipDiv()) {
+            if (this._isTooltipAlwaysVisible()) {
+                this._showHideTooltipDiv(true);
+            } else {
+                this.widget.addEventListener("mouseover", () => {
+                    this._showHideTooltip( true);
+                });
+                this.widget.addEventListener("mouseout", () => {
+                    this._showHideTooltip(false);
+                });
+            }
+        }
+        if (this.getDescription()) {
+            this._addHelpIconHandler();
+        }
+    }
+
+    /**
+     *
+     * @param show If true then <div> containing tooltip(Short Description) will be shown else hidden
+     * @private
+     */
+    _showHideTooltipDiv(show) {
+        this._toggleAttribute(this.getTooltipDiv(), show, this.dataAttribute('shortdescription-hidden'), true);
+    }
+
+    /**
+     *
+     * @param show If true then <div> containing description(Long Description) will be shown
+     * @private
+     */
+    _showHideLongDescriptionDiv(show) {
+        this._toggleAttribute(this.getDescription(), show, this.dataAttribute('longdescription-hidden'), true);
+    }
+
+    _isTooltipAlwaysVisible() {
+        const state = this._model.getState();
+        return state && state.properties && state.properties['af:layout'] && state.properties['af:layout'].tooltipVisible;
```

Comment:
@rismehta We are using `afs` and `af` and `fd`. I think `afs` and `fd` should be there

Reply:
Yes, this should be afs:layout. Can you change this @barshat7